opengrok-web/src/main/java/org/opengrok/web/messages/MessagesContainer.java

```diff
@@ -51,14 +53,16 @@
 import java.util.logging.Logger;
 import java.util.stream.Collectors;
 
-public class MessagesContainer {
+public class MessagesContainer implements OnConfigurationChangedListener {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(MessagesContainer.class);
 
     public static final String MESSAGES_MAIN_PAGE_TAG = "main";
 
     private static final int DEFAULT_MESSAGE_LIMIT = 100;
 
+    private static final MessagesContainer instance = new MessagesContainer();
```

Comment:
capital letters

Reply:
Actually no, this is not a constant in a matter of speaking and capital letters should not be used. I already posted the java language specification regarding this to @vladak once.

Reply:
2 lines above the naming is consistently with capital letters